airflow-core/src/airflow/config_templates/config.yml

```
@@ -2509,6 +2509,17 @@ dag_processor:
       type: integer
       example: ~
       default: "30"
+    dag_processor_health_check_threshold:
```

```suggestion
    health_check_threshold:
```
This is already under the dag_processor section. I guess you want it to align with scheduler /triggerer but I think it's ok to have it as health_check_threshold only

---

Thank you for the review @ephraimbuddy , I will address it

---

addressed comments